apps/vr-tests-react-components/src/stories/MessageBar/MessageBar.stories.tsx

```diff
@@ -64,6 +64,27 @@ export const Multiline = () => (
   </div>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30481&commitId=572aaf723d6321e59014c3388cdf419eaf5497a0&env=prod&clientType=FLUENTUI)


  <details><summary><b>MessageBar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Multiline No Actions.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>